multihost_test/Sanity/test_pam_pwquality.py

```diff
@@ -0,0 +1,149 @@
+        multihost.client[0].transport.put_file(os.getcwd() +
+                                               '/script/pam_pwquality.sh',
+                                               '/tmp/pam_pwquality.sh')
+        exceute_cmd(multihost, "chmod 755 /tmp/pam_pwquality.sh")
+        exceute_cmd(multihost, "sh /tmp/pam_pwquality.sh  "
+                               "local_anuj pass R3dh4T1nC R3dh4T1nC > /tmp/anuj")
+        assert "New PAMTEST password" in exceute_cmd(multihost,
+                                                     "cat /tmp/anuj").stdout_text
+        assert "Retype new PAMTEST password" in exceute_cmd(multihost,
+                                                            "cat /tmp/anuj").stdout_text
+        exceute_cmd(multihost, "sed -i 's/.*pam_pwquality.*/password   "
+                               "requisite pam_pwquality.so retry=1 enforce_for_root/'  "
+                               "/etc/pam.d/system-auth")
+        with pytest.raises(subprocess.CalledProcessError):
+            exceute_cmd(multihost, "sh /tmp/pam_pwquality.sh  "
+                                   "local_anuj R3dh4T1nC k4ddL32rfg NOPASS")
+        exceute_cmd(multihost, "sed -i 's/.*pam_pwquality.*/password   "
+                               "requisite pam_pwquality.so retry=2 enforce_for_root/'  "
+                               "/etc/pam.d/system-auth")
+        with pytest.raises(subprocess.CalledProcessError):
+            exceute_cmd(multihost, "sh /tmp/pam_pwquality.sh  "
+                                   "local_anuj R3dh4T1nC k4ddL32rfg NOPASS")
+        exceute_cmd(multihost, "sed -i 's/.*pam_pwquality.*/password   "
+                               "requisite pam_pwquality.so retry=1 difok=3 enforce_for_root/'  "
+                               "/etc/pam.d/system-auth")
+        with pytest.raises(subprocess.CalledProcessError):
+            exceute_cmd(multihost, "sh /tmp/pam_pwquality.sh  "
+                                   "local_anuj R3dh4T1nC aR3dh4T1nCb aR3dh4T1nCb")
+        exceute_cmd(multihost, "sh /tmp/pam_pwquality.sh  "
+                               "local_anuj R3dh4T1nC aR3dh4T1nCbD aR3dh4T1nCbD")
+        exceute_cmd(multihost, "sed -i 's/.*pam_pwquality.*/password   "
+                               "requisite pam_pwquality.so retry=1 minlen=5 enforce_for_root/'  "
+                               "/etc/pam.d/system-auth")
+        exceute_cmd(multihost, "sh /tmp/pam_pwquality.sh  "
+                               "local_anuj aR3dh4T1nCbD jf@#FafR3dh4T1nC!!F jf@#FafR3dh4T1nC!!F")
+        with pytest.raises(subprocess.CalledProcessError):
+            exceute_cmd(multihost, "sh /tmp/pam_pwquality.sh  "
+                                   "local_anuj jf@#FafR3dh4T1nC!!F 3214 3214")
+        exceute_cmd(multihost, "cp -vf /etc/pam.d/system-auth_anuj  "
+                               "/etc/pam.d/system-auth")
+        exceute_cmd(multihost, "useradd pamtest1")
```

Comment:
This can be a different test case with its own test case title and id. Above test case was len and credit related. This one is for gecoscheck

Reply:
DOne